demo/schemas/user-collection.js

```diff
@@ -7,6 +7,17 @@ module.exports = {
     description: 'A collection of users.',
     type: 'object',
     properties: {
+        limit: {
```

Comment:
I agree with the default, but it adds a little more complexity to the way `Request` objects are handled. There's already complexity involving converting the values to ints/floats as necessary, which is creating a tighter coupling to the schema/route definition than I'd like. But I'll look into this.